source/common/runtime/runtime_impl.h

```diff
@@ -34,6 +34,9 @@ class RandomGeneratorImpl : public RandomGenerator {
   uint64_t random() override { return threadLocalGenerator()(); }
   std::string uuid() override;
 
+  // Runtime::RandomGeneratorImpl
```

Comment:
nit: del comment, move randomBytes decl up above line 33. (interface overrides come last in visibility section so it's obvious what is owned by what, we don't have comment for normal defines).